swarm/api/api_test.go

```diff
@@ -433,3 +433,52 @@ func TestDecryptOrigin(t *testing.T) {
 		}
 	}
 }
+
+func TestDetectContentType(t *testing.T) {
+	// internally use http.DetectContentType, so here are test cases only about fallback to file extension check
+	testDetectContentType(t, "file.css", "body {background-color: orange}", "text/css; charset=utf-8")
```

Comment:
This should use [`t.Run`](https://golang.org/pkg/testing/#T.Run) to isolate each test.